### ISSUE TITLE
[ESLint] Support YAML configuration files

### ISSUE THREAD

**Issue by Hirse** [angelozerr/tern.java#423]
ESLint itself accepts configuration files in [several formats](http://eslint.org/docs/user-guide/configuring#configuration-file-formats).

Is it possible to enable usage of YAML files which are passed to ESLint?

**Comment by angelozerr** [comment]
Never tested, have you tried?

**Comment by Hirse** [comment]
Not personally, but one colleague at work tried and was unable to use `.yml` files.
Only after converting our rule-file to json, we were able to use it.

**Comment by angelozerr** [comment]
It's very difficult for me to find time to support that, any contribution are welcome!

**Comment by bobkubista** [comment]
I would also like javascript files to be supported. Should I open a new issue for this? Tern doesn't accept the config file at the moment.
